### ISSUE TITLE
Date picker improvements - ISO8601 support, add more presets, UI rearrangement, time picker

### ISSUE THREAD

**Issue by C0rn3j** [wekan/wekan#5445]
Current             |  Suggested | Alternative with preset on top
:-------------------------:|:-------------------------:|:-------------------------:
![](https://github.com/wekan/wekan/assets/1641362/07736011-1057-4861-9f31-9099913f8d19)  |  ![](https://github.com/wekan/wekan/assets/1641362/2333c362-0001-4aa2-bfaf-a3dd62feedd6) | ![](https://github.com/wekan/wekan/assets/1641362/201a5482-85a8-4288-bab3-05d87534735d)

I have a couple suggestions for the date picker.

- [ ] First would be support in the settings to optionally use ISO8601 over local time.

- [ ] Second would be to rearrange the elements to make them flow and look a bit smoother

- [ ] Third would be to add more delay presets than just "Today", and possibly make them into buttons/icons.

- [ ] Fourth would be to implement a time picker, to allow to define preset times without the keyboard, like so:  
![image](https://github.com/wekan/wekan/assets/1641362/0823260d-ccad-4f25-983c-bbaeaf02de2b)

**Comment by xet7** [comment]
@C0rn3j 

Some improvements to date picker would be nice, like using keyboard arrow keys, tab etc to set time.

Also, we would need some way to select exact time by only clicking, because writing is difficult with some keyboards of some languages #3873 

There is also some other problems with dates #4881 

But only changing place of date and time would just get people confused.

**Comment by Hekatomb** [comment]
> @C0rn3j
> 
> Some improvements to date picker would be nice, like using keyboard arrow keys, tab etc to set time.
> 
> Also, we would need some way to select exact time by only clicking, because writing is difficult with some keyboards of some languages #3873
> There is also some other problems with dates #4881
> 
> But only changing place of date and time would just get people confused.

Is there reason you are not using the browser's built-in time picker? Looks like it is well supported in mobile browsers and desktop browsers alike.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/time
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date

**Comment by xet7** [comment]
@Hekatomb 

It looks like week number is not visible in browser's built-in time picker in Firefox.

**Comment by Hekatomb** [comment]
@xet7  That is correct and that could very well be a reason not to use the built-in date picker. But you can use the time picker without the date picker.

**Comment by xet7** [comment]
WeKan now uses browser native date and time pickers.
